src/game/engine/gametypes.h

```diff
@@ -746,6 +744,9 @@ enum SpecialWeaponType {
     SPECIAL_IRON_CURTAIN,
     SPECIAL_GPS,
     SPECIAL_COUNT,
+    SPECIAL_WARP_TELEPORT = SPECIAL_COUNT,
```

Comment:
Can we add a comment here so it explains why SpecialWeaponType is using a different format compared all other enum types.